extension/pybindings/portable_lib.py

```diff
@@ -29,6 +29,15 @@
     ExecuTorchModule,  # noqa: F401
 )
 
+ try:
+     from pathlib import Path
+     libs = list(Path(__file__).parent.parent.parent.resolve().glob("**/libquantized_ops_aot_lib.*"))
+     del Path
+     assert len(libs) == 1, f"Expected 1 library but got {len(libs)}"
+     _torch.ops.load_library(libs[0])
+ except:
+     pass
+
```

Comment:
So we probably don't want this code to live here. We should load it whenever we need it. I think we should add this code in __init__.py under this [directory](https://github.com/pytorch/executorch/tree/main/kernels/quantized), so that you can import it in the unittest [here](https://github.com/pytorch/executorch/blob/main/kernels/quantized/test/test_out_variants.py)